service/integration/module.go

```
@@ -0,0 +1,48 @@
+
+import (
+	"github.com/safing/portmaster/service/mgr"
+	"github.com/safing/portmaster/service/updates"
+)
+
+// OSIntegration module provides special integration with the OS.
+type OSIntegration struct {
+	m      *mgr.Manager
+	states *mgr.StateMgr
+
+	os OSSpecific
```

golangci-lint

field `os` is unused

(unused)